### PR TITLE
modify the date validation

### PR DESCRIPTION
**Pull request by samudra-ajri**
Modify the date validation for the Command Center reservation closed dates.
- date must be unique
- admin could not input the passed days

@jabardigitalservice/jds-backend

### CODE REVIEW

File: app/Http/Requests/UpdateCommandCenterCloseDateRequest.php [hunk 1]
```diff
@@ -24,7 +24,7 @@ public function authorize()
     public function rules()
     {
         return [
-            'date' => 'date',
+            'date' => 'date|after_or_equal:today|unique:command_center_close_dates',
```
Comment (iprasetyaa) [inline app/Http/Requests/UpdateCommandCenterCloseDateRequest.php:27]: pas update dia ga required jri?

Comment (samudra-ajri) [reply app/Http/Requests/UpdateCommandCenterCloseDateRequest.php:27]: Ngga, Kang..
Jd pas update, klw dia ngga ngisi auto pakai value yg sebelumnya, Kang

Comment (iprasetyaa) [reply app/Http/Requests/UpdateCommandCenterCloseDateRequest.php:27]: Oke jri mantap

Comment (iprasetyaa) [reply app/Http/Requests/UpdateCommandCenterCloseDateRequest.php:27]: oh i see, siap kalau begitu

**Review by rachadiannovansyah — APPROVED** [review]

**Review by iprasetyaa — APPROVED** [review]

**Pull request merged**